apps/intellij/src/main/kotlin/dev/nx/console/Icons.kt

```diff
@@ -3,6 +3,6 @@ package dev.nx.console
 import com.intellij.openapi.util.IconLoader
 
 object NxIcons {
-    @JvmField val Action = IconLoader.getIcon("icons/nx-icon.png", javaClass)
-    @JvmField val FileType = IconLoader.getIcon("icons/nx-icon.png", javaClass)
+    @JvmField val Action = IconLoader.getIcon("icons/nx-console_dark.svg", javaClass)
```

Comment:
I think we should specify icons without dark suffixe

<img width="621" alt="Screenshot 2023-02-23 at 07 49 20" src="https://user-images.githubusercontent.com/9085108/220837659-f69b82de-aa93-4287-8b4a-d4f825068979.png">

Reply:
Ah damn you're right. I don't know how that slipped through.

Reply:
I know who it slipped through, I renamed the file and intellij updated the name here 😅